src/test/java/uk/gov/digital/ho/pttg/application/namematching/InputNamesTest.java

```diff
@@ -0,0 +1,124 @@
+package uk.gov.digital.ho.pttg.application.namematching;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class InputNamesTest {
+
+    @Test
+    public void multiPartLastNameEmpty() {
+        InputNames inputNames = new InputNames("any-name", "");
+
+        assertThat(inputNames.multiPartLastName()).isFalse();
+    }
+
+    @Test
+    public void multiPartLastNameSingleName() {
+        InputNames inputNames = new InputNames("any-name", "any-lastname");
+
+        assertThat(inputNames.multiPartLastName()).isFalse();
+    }
+
+    @Test
+    public void multiPartLastNameSingleTwoNames() {
+        InputNames inputNames = new InputNames("any-name", "lastname1 lastname2");
+
+        assertThat(inputNames.multiPartLastName()).isTrue();
+    }
+
+    @Test
+    public void multiPartLastNameFirstNameIIrrelevant() {
```

Comment:
I think theres an S missing? multiPartLastNameFirstNameI(s)Irrelevan not multiPartLastNameFirstNameIIrrelevant?